src/app/modules/main/wallet_section/send/network_route_model.nim

```diff
@@ -26,49 +19,43 @@ type
     ToNetworks
 
 QtObject:
-  type NetworkModel* = ref object of QAbstractListModel
-    items*: seq[NetworkItem]
+  type NetworkRouteModel* = ref object of QAbstractListModel
+    delegate: io_interface.AccessInterface
```

Comment:
@Cuteivist a reference to the module should not be passed to the model, that's too much, this way the model is too coupled to the module. Please remove it and pass a list of networkItem or map of [chainId, data-you-need].